build/charts/flow-aggregator/values.yaml

```diff
@@ -84,6 +84,31 @@ s3Uploader:
     aws_access_key_id: "changeme"
     aws_secret_access_key: "changeme"
     aws_session_token: ""
+# flowLogger contains configuration options for writing flow records to a local log file.
+flowLogger:
+  # -- Determine whether to enable exporting flow records to a local log file.
+  enable: false
+  # -- Path is the path to the local log file.
+  path: "/tmp/antrea-flows.log"
+  # -- MaxSize is the maximum size in MB of a log file before it gets rotated.
+  maxSize: 100
+  # -- MaxBackups is the maximum number of old log files to retain. If set to 0, all log files will
+  # be retained (unless MaxAge causes them to be deleted).
+  maxBackups: 3
+  # -- MaxAge is the maximum number of days to retain old log files based on the timestamp encoded
+  # in their filename. The default (0) is not to remove old log files based on age.
+  maxAge: 0
+  # -- Compress enables gzip compression on rotated files.
+  compress: true
+  # -- RecordFormat defines the format of the flow records logged to file. Only "CSV" is supported at the moment.
+  recordFormat: "CSV"
+  # -- Filters can be used to select which flow records to log to file. The provided filters are
+  # OR-ed to determine whether a specific flow should be logged. By default, all flows are logged.
+  filters: []
+  # With the following filters, only flow which are denied because of a network policy will be logged.
+  # filters: [{ingressNetworkPolicyRuleActions: ["Drop", "Reject"]}, {egressNetworkPolicyRuleActions: ["Drop", "Reject"]}]
```

Comment:
Suggest moving line 107 after this line of comment.

Reply:
I moved it, and re-organized the comment a bit so that it doesn't look out-of-place in the generated README

Reply:
Done